.circleci/config.yml

```diff
@@ -0,0 +1,97 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_EMAIL   - login info for docker hub
```

Comment:
DOCKER_EMAIL is no longer needed. This would also need to be updated https://github.com/mozilla/missioncontrol/blob/master/bin/deploy#L25